pkg/model/model.go

```diff
@@ -65,10 +65,11 @@ func (fme FunctionMaskContextEngine) MaskContext(e Dictionary, key string, conte
 }
 
 type MaskFactoryConfiguration struct {
-	Masking   Masking
-	Seed      int64
-	Cache     map[string]Cache
-	Functions tmpl.FuncMap
+	Masking       Masking
+	Seed          int64
+	SeedFromClock bool
```

Comment:
Why the mask factory have to know that the seed come from clock and not from configuration ?

Reply:
Because the seed is set once and for all at the beginning of the program, and I need to know the difference between two case :

1. the seed has a value because it was not set and then it was given a value generated from clock
2. the seed has a value because the user gave it

In case 1. we will never use the value of the seed to propagate it in the `seed` parameter of a mask - because the user expect coherence in each generation regarding the original value

In case 2. we will use the value set by user and propagate it in the `seed` parameter of each mask (sum of both seed) because the user expect the coherence to change if the global seed is changed.